src/app/zap-templates/zcl/operational-credentials-cluster.xml

```diff
@@ -59,5 +59,10 @@ fabric-scoped data.</description>
       <arg name="FabricId" type="FABRIC_ID"/>
     </command>
 
+    // Up for discussion in Multi-Admin TT: chip-spec:#2891
+    <command source="client" code="0x02" name="RemoveAllFabrics" optional="false">
```

Comment:
Could you give it a very different ID until we know the ID? Otherwise there is even more changes to be done when we make Opcreds cluster spec compliant.

Reply:
Added it ID 11 which is unused in the spec (last one is 10 for RemoveFabric)